src/libtrx/include/libtrx/game/game_string.def

```diff
@@ -398,3 +398,128 @@ GS_DEFINE(GRAPHIC_SETTINGS_ENABLE_ROUND_SHADOW,        "Round shadows")
+GS_DEFINE(GAMEPLAY_SETTINGS_FIX_DESCENDING_GLITCH_DESCRIPTION,          "Fixes sidestepping and walking backwards on breakable tiles causing Lara to immediately descend to the tile underneath.")
+GS_DEFINE(GAMEPLAY_SETTINGS_FIX_TEXTURE_ISSUES_DESCRIPTION,             "Fixes original issues with missing or incorrect textures/meshes.")
+GS_DEFINE(GAMEPLAY_SETTINGS_FIX_FREE_FLARE_GLITCH_DESCRIPTION,          "Fixes the ability to spawn a free flare when pressing the flare input while picking up any item.")
+GS_DEFINE(GAMEPLAY_SETTINGS_FIX_FLARE_THROW_PRIORITY_DESCRIPTION,       "Fixes Lara prioritising throwing a spent flare while in mid-air, which can lead to being unable to grab ledges.")
+GS_DEFINE(GAMEPLAY_SETTINGS_FIX_WALK_RUN_JUMP_DESCRIPTION,              "Fixes Lara at times not being able to jump immediately after going from her walking to running animation.")
+GS_DEFINE(GAMEPLAY_SETTINGS_FIX_GLIDE_CAMERAS_DESCRIPTION,              "Fixes glide cameras to respect the timer set in level triggers rather than defaulting the speed after the first frame.")
+GS_DEFINE(GAMEPLAY_SETTINGS_FIX_ITEM_DUPLICATION_GLITCH_DESCRIPTION,    "Fixes the ability to duplicate usage of key items in the inventory.")
+GS_DEFINE(GAMEPLAY_SETTINGS_FIX_ITEM_ROTS_DESCRIPTION,                  "Fixes original issues with some incorrectly rotated pickups when using the 3D pickups option.")
+GS_DEFINE(GAMEPLAY_SETTINGS_FIX_M16_ACCURACY_DESCRIPTION,               "Fixes the accuracy of the M16 while Lara is running.")
+GS_DEFINE(GAMEPLAY_SETTINGS_FIX_PICKUP_DRIFT_GLITCH_DESCRIPTION,        "Fixes Lara sometimes drifting when collecting underwater items, which can push her into walls.")
+GS_DEFINE(GAMEPLAY_SETTINGS_FIX_QWOP_GLITCH_DESCRIPTION,                "Fixes Lara jumping on small steps sometimes resulting in a weird running animation, known as a QWOP state.")
+GS_DEFINE(GAMEPLAY_SETTINGS_FIX_STEP_GLITCH_DESCRIPTION,                "Fixes Lara sometimes being pushed into walls adjacent to steps when running up them in a specific way.")
+GS_DEFINE(GAMEPLAY_SETTINGS_FIX_TIHOCAN_SECRET_SOUND_DESCRIPTION,       "Prevents the secret sound from incorrectly playing when using the golden key in Tomb of Tihocan.")
+GS_DEFINE(GAMEPLAY_SETTINGS_FIX_WALL_JUMP_GLITCH_DESCRIPTION,           "Fixes Lara jumping through objects and moving up certain slopes.")
+GS_DEFINE(GAMEPLAY_SETTINGS_LOAD_MUSIC_TRIGGERS_DESCRIPTION,            "Loads previously triggered, one shot music so one shot music tracks do not replay.")
+GS_DEFINE(GAMEPLAY_SETTINGS_MAXIMUM_SAVE_SLOTS_DESCRIPTION,             "Changes the number of available save slots.")
+GS_DEFINE(GAMEPLAY_SETTINGS_REMEMBER_GUNS_BETWEEN_LEVELS_DESCRIPTION,   "Makes Lara revert to pistols at the start of each level.")
```

Comment:
Perhaps wording here would be better as follows now that it's inverted (I forgot about this earlier, apologies).
`Makes Lara remember which gun she was using last in the previous level when starting a new level. If disabled, Lara will revert to pistols.`

Reply:
Thanks, updated